index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+      </style>
+    <title>Document</title>
+    
+</head>
+<body>
+    <header class="navegation-menu">
+    <div class="humburguer"><i href="#" class="fa fa-bars"></i>
+    </div> 
+    </header>
+
+    <main>  
+        <section class="headline-section">
+
+
+            <div class="headline-container">
+                <div class="header-text">
+                    <h1>Hey there.</h1>
+                    <h1>I'm Graciano.</h1>
```

Comment:
- [ ] For good semantics and accessibility it would be better if you use just one `h1` tag on each page as it's the title header of your page.